app/uk/gov/hmrc/teamsandservices/TeamsServicesController.scala

```diff
@@ -72,7 +86,8 @@ trait TeamsServicesController extends BaseController {
       val cached = teams.asTeamServices(teamName, ciUrlTemplates)
       cached.data match {
         case Nil => NotFound
-        case _ => Ok(Json.toJson(cached))
+        case _ => Ok(Json.toJson(cached.data))
```

Comment:
It might be nice to have some kind of helper method that will form the Ok response, take care of the Json and add the header all in one go, will make the controller code much nicer and less verbose.